rust/datafusion/src/execution/physical_plan/mod.rs

```diff
@@ -74,6 +85,15 @@ impl Partitioning {
     }
 }
 
+/// Distribution schemes
+#[derive(Debug, Clone)]
+pub enum Distribution {
+    /// Unspecified distribution
```

Comment:
Maybe `/// no distribution is required`?

Reply:
I think "unspecified" is more accurate -- specifically the data is distributed around all the partitions, but the optimizer/planner doesn't know *how* it is distributed